src/main/java/dev/ebullient/convert/tools/dnd5e/Tools5eJsonSourceCopier.java

```diff
@@ -196,48 +196,57 @@ public JsonNode mergeSubrace(JsonNode subraceNode, JsonNode raceNode) {
         return subraceOut;
     }
 
-    // 	utils.js: static getCopy (impl, copyFrom, copyTo, templateData,...) {
     @Override
-    protected JsonNode mergeNodes(Tools5eIndexType type, String originKey, JsonNode copyFrom, JsonNode copyTo) {
-        // edit in place: if you don't, lower-level copies will keep being revisted.
-        ObjectNode target = (ObjectNode) copyTo;
-
-        JsonNode _copy = MetaFields._copy.getFrom(copyTo);
-        JsonNode _mod = _copy == null ? null : normalizeMods(MetaFields._mod.getFrom(_copy));
-        JsonNode _trait = _copy == null ? null : MetaFields._trait.getFrom(_copy);
-
+    protected JsonNode handleTrait(JsonNode copyInfo) {
```

Comment:
Not something that can be put in a method. Manipulation of local state in layers before it is applied to the object does actually matter.

Reply:
I'll leave this for now to be handled in a separate PR with the rest of the _trait (now _template) stuff. I expect that this will have to be approached differently anyway, and I'll need to see if it is actually used on the pf2e side at all.